### ISSUE TITLE
Bad Interpreter

### ISSUE THREAD

**Issue by james-nesbitt** [seth-griffin/MultiGet#2]
Can you improve your Spike app for this error please:

```
:/app/sgriffin# ./spike --url=http://6f5a196.bwtest-aws.pravala.com/384MB.jar --outputFileName=4MiB
bash: ./spike: /usr/bin/php: bad interpreter: No such file or directory
```

**Comment by seth-griffin** [comment]
Hey James. What OS and version are you using? Which PHP version?

**Comment by seth-griffin** [comment]
Hi James,

This should be fixed as of e8c17d19294eaa1b6d0324f76c45d3cc0efffb59. Please try that commit to see if deferring to /usr/bin/env php did the trick.

**Comment by seth-griffin** [comment]
Reopening so that James can verify that this is fixed.
